partner_firstname/models/partner.py

```diff
@@ -1,26 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
This is a guideline, and yes, it's technical :wink: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming